packages/grafana-e2e-selectors/src/selectors/pages.ts

```diff
@@ -62,7 +62,6 @@ export const Pages = {
       Annotations: {
         List: {
           addAnnotationCTA: Components.CallToActionCard.button('Add annotation query'),
-          newButton: 'Annotations list new annotation button',
```

Comment:
Hmm, not sure we should ever remove a selector like this. This will make older versions of tests break, right?

Reply:
This was a selector I initially added for this PR 😄 
I've been wondering if we should be using `aria-labels` for elements like this where we have text visible

`Use it in cases where a text label is not visible on the screen`:
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-label_attribute

I think this topic has been added to the next front end weekly agenda. Thoughts?

Reply:
Ok my bad, you're correct!